includes/Admin/Orders.php

```diff
@@ -306,9 +306,15 @@ public function maybe_stop_order_email( $is_enabled, $order ) {
 	 * @param \WC_Order $order order object
 	 * @return bool
 	 */
-	public function is_order_editable( $maybe_editable, \WC_Order $order ) {
+	public function is_order_editable( $maybe_editable, $order ) {
 
-		return $maybe_editable;
+		// checks if the order is an instance of WC_Order
+		$is_wc_order = isset( $order ) && $order instanceof \WC_Order;
```

Comment:
@adrianostanley my only question is the purpose of the `isset()` here - `$order` will always be set (even if null) by virtue of being a known param, no?

Reply:
@ChaseWiseman correct. I used an `isset` before calling `Commerce\Orders::is_order_pending( $order )` and then probably didn't pay a good attention when refactoring to check if it's an instance of `\WC_Order`. That should be fixed in 5f823fce1980c900b4d1632736cac3db04476cae.